scripts/build_all_chainspecs.sh

```diff
@@ -55,8 +69,8 @@ update_spec() {
 
   echo "*** Restoring genesis in '$chain'..."
 
-  update_genesis "$raw_genesis_temp" "$raw_spec_temp" "$raw_path"
-  update_genesis "$plain_genesis_temp" "$plain_spec_temp" "$plain_path"
+  update_genesis_and_code_substitutes "$raw_genesis_temp" "$raw_code_substitutes_temp" "$raw_spec_temp" "$raw_path"
+  update_genesis_and_code_substitutes "$plain_genesis_temp" "$plain_code_substitutes_temp" "$plain_spec_temp" "$plain_path"
 
   # cleanup
   rm -f "$raw_genesis_temp" "$plain_genesis_temp" "$raw_spec_temp" \
```

Comment:
Makes sense adding `raw_code_substitutes_temp` and `plain_code_substitutes_temp` here as well.